### DISCUSSION TITLE
LDAP Authentication / Windows AD

### DISCUSSION THREAD

**Discussion by willflo** [netbox-community/netbox#8699] [Q&A]
Hey everybody, 

We are currently using LDAP Authentication via Windows Active Directory Groups. We are just using those Django standard Flags. "is_active", "is_superuser", "is_staff".
Now I want to create a whole Group Structure for LDAP Authentication in Netbox. The Groups in Active Directory are already there but I just dont know how to implement all these groups into the Django Authentication for LDAP. Is it enough if I just create the Groups via the Staff Page? Do i need the add them to the LDAP Config and if yes, how? I am totally lost.

I know this isnt the typical Netbox Question but i would be happy if someone could help me with this.

Thank you so much.

**Comment by jhammond-git** [comment]
You may be looking for AUTH_LDAP_MIRROR_GROUPS.

**Comment by jhammond-git** [comment]
But you'll probably want to read this: https://django-auth-ldap.readthedocs.io/en/latest/permissions.html
